mslib/utils/airdata.py

```diff
@@ -79,19 +80,24 @@ def download_progress(file_path, url, progress_callback=lambda f: logging.info(f
         QtWidgets.QMessageBox.information(None, "Download failed", f"{url} was unreachable, please try again later.")
 
 
-def get_airports(force_download=False):
+def get_airports(force_download=False, url=None):
     """
     Gets or downloads the airports.csv in ~/.config/msui and returns all airports within
     """
     global _airports, _airports_mtime
-    file_exists = os.path.exists(os.path.join(MSUI_CONFIG_PATH, "airports.csv"))
+    if url is None:
+        url = "https://ourairports.com/data/airports.csv"
+    data = fs.open_fs(MSUI_CONFIG_PATH)
+    osdir = data.root_path
+
+    file_exists = os.path.exists(os.path.join(osdir, "airports.csv"))
```

Comment:
is there a good reason not to store the joined path in a variable? This repetion is (as demonstrated here) tedious.

Reply:
of course no good reason, :)